phoenix-core/src/it/java/org/apache/phoenix/end2end/SortMergeJoinIT.java

```diff
@@ -187,13 +186,13 @@ public void initTable() throws Exception {
                 "CREATE LOCAL INDEX \"idx_supplier\" ON " + JOIN_SUPPLIER_TABLE_FULL_NAME + " (name)"
                 }, {
                 "SORT-MERGE-JOIN (LEFT) TABLES\n" +
-                "    CLIENT PARALLEL 1-WAY RANGE SCAN OVER " + MetaDataUtil.LOCAL_INDEX_TABLE_PREFIX + JOIN_SUPPLIER_TABLE_DISPLAY_NAME + " [-32768]\n" +
+                "    CLIENT PARALLEL 1-WAY RANGE SCAN OVER " +JOIN_SUPPLIER_TABLE_DISPLAY_NAME + " [1]\n" +
```

Comment:
Yes, I think it'd be good to mention the local index name as there could be more than one. If you include the full name of the local index (<schema name>.<table name> which is what you get from PTable.getName()), then that's unique so you wouldn't need to include the data table name too.